build.gradle.kts

```diff
@@ -411,12 +407,8 @@ subprojects {
     reports.html.outputLocation.set(file("${rootProject.projectDir}/build/reports/"))
     val skipTests = project.hasProperty("skipTests")
     if (!skipTests) {
-      if (project.name == "trino-connector") {
-        useTestNG()
-        maxHeapSize = "2G"
-      } else {
-        useJUnitPlatform()
-      }
+      maxHeapSize = "2G"
```

Comment:
Is `maxHeapSize` still needed for Junit?

Reply:
I'm not sure, I remove it first